wiki/osu!_wiki/Contribution_guide/th.md

```diff
@@ -0,0 +1,67 @@
+
+## การแก้ไข wiki
+
+สำหรับกระบวนการการมีส่วนร่วมทั่วไป ลองตรวจสอบ [GitHub flow - GitHub Docs](https://docs.github.com/en/get-started/quickstart/github-flow)*
+
+บทความของ osu! wiki ถูกบันทึกอยู่ใน [GitHub][osu_wiki] หากต้องการช่วยเหลือ ทำตามขั้นตอนเหล่านี้
+
+0. [สร้าง](https://github.com/signup) บัญชี GitHub
+1. เปิด [`ppy/osu-wiki`][osu_wiki] และเลือก `Fork` จากมุมขวาบน เพื่อให้สามารถควบคุม wiki ฉบับคัดลอก คุณสามารถเลือกไปที่ fork ของตัวเองได้โดยการไปที่ [`ppy/osu-wiki`][osu_wiki] และเลือก `Fork` อีกครั้ง
+
+  - ถ้าคุณได้สร้าง fork เมื่อนานมาแล้ว สามารถซิงค์ได้โดยวิธีการนี้ [Best practices § Syncing the fork](/wiki/osu!_wiki/Contribution_guide/Best_practices#syncing-the-fork)
+
+2. อ่าน [Best practices § Making changes](/wiki/osu!_wiki/Contribution_guide/Best_practices#making-edits) และเริ่มดำเนินการที่จำเป็นได้เลยโดยคุณมีอิสระในการเลือกแอปพลิเคชั่นในการทำ อย่างไรก็ตาม osu! wiki มีคำแนะนำโดยละเอียดสำหรับการใช้ 2 อย่างนี้
+
+  - [GitHub web-based editor](/wiki/osu!_wiki/Contribution_guide/GitHub_web-based_editor) (ออนไลน์, ไม่มีการติดตั้งเพิ่มเติม).
+  - [GitHub Desktop](/wiki/osu!_wiki/Contribution_guide/GitHub_Desktop) (ออฟไลน์, สามารถจัดการอะไรได้มากกว่า).
+
+## ตรวจสอบงานของตัวเอง
+
+เมื่อคุณได้สร้างการเปลี่ยนแปลงใด ๆ เรียบร้อยแล้ว โปรดใช้เวลาในการอ่านทบทวนงานของคุณ โดยตรวจสอบตามขั้นตอนเหล่านี้
+
+- **น้ำเสียงของบทความ**: บทความของ osu! wiki จะต้องถูกเขียนโดยการใช้ความเป็นกลาง โดยมีข้อยกเว้นเล็กน้อยในบางกรณี 
+- **สไตล์ และไวยากรณ์**: บทความต่าง ๆ ต้องมีความชัดเจน เข้าใจง่าย ไม่ส่งผลกระทบต่อจิตใจผู้อ่าน เนื้อหามีความสอดคล้อง และหลีกเลี่ยงประโยคที่มีความซับซ้อน คุณสามารถใช้ตัวช่วยเขียนที่มีตัวตรวจคำผิดอย่าง [Google Docs](https://docs.google.com) เผื่อแก้ไขคำผิด ไวยากรณ์ และการใช้ syntax ต่าง ๆ 
+- **[Content parity](/wiki/Article_styling_criteria/Writing#content-parity)** บทแปลจะต้องมีเนื้อหาเดิมกับบทความต้นฉบับ (การเปลี่ยนแปลงทางด้านการเว้นวรรค การใช้คำ หรือการจัดรูปแบบ เป็นสิ่งที่เกิดขึ้นได้) แทนที่จะเปลี่ยนเนื้อหาบทความต้นฉบับให้ลอง [open an issue](https://github.com/ppy/osu-wiki/issues/new) หรือเปิด pull request อันใหม่ที่แก้ไขบทความต้นฉบับ ถ้าคุณรู้สึกว่ามันยังไม่สมบูรณ์ ไม่ถูกต้อง หรือเก่าเกินไป
+- **โครงสร้าง และการจัดรูปแบบ** ลองดูตัวอย่างงานของคุณผ่านเครื่องมืออย่าง [jbt's Markdown Editor](https://jbt.github.io/markdown-editor/) เพื่อทำให้แน่ใจว่างานของคุณมีหน้าตาเหมือนกับที่คุณคิดไว้
+- **รูปภาพทุกรูป และไฟล์อื่น ๆ ที่ไม่ใช้ตัวหนังสือ** จะต้อง [เล็กกว่า 1 megabyte](/wiki/Article_styling_criteria/Formatting#file-size) **ภาพหน้าจอ** จะต้องใช้สกินเริ่มต้นของ osu! และใช้ [การตั้งค่าที่กำหนด](/wiki/Article_styling_criteria/Formatting#screenshots-of-gameplay) โดยมีขนาดไม่เกิน 1280×720
+
+สำหรับมาตราฐานการเขียน และการจัดรูปแบบ สามารถดูได้ที่ [เกณฑ์การจัดรูปแบบบทความ](/wiki/Article_styling_criteria) โดยไม่แนะนำให้อ่านตั้งแต่ต้นจนจบในครั้งเดียว และผู้ตรวจสอบจะแจ้งจุดที่ผิดพลาดและต้องแก้ไขหากพบเจอ
```

Comment:
A bit typo here

```suggestion
สำหรับมาตรฐานการเขียน และการจัดรูปแบบ สามารถดูได้ที่ [เกณฑ์การจัดรูปแบบบทความ](/wiki/Article_styling_criteria) โดยไม่แนะนำให้อ่านตั้งแต่ต้นจนจบในครั้งเดียว และผู้ตรวจสอบจะแจ้งจุดที่ผิดพลาดและต้องแก้ไขหากพบเจอ
```

Reply:
Thanks for the review! Sorry for the typo and wrong translation, fixed